apps/backoffice-v2/src/domains/business-reports/fetchers.ts

```diff
@@ -43,6 +44,10 @@ export const BusinessReportSchema = z
   })
   .transform(data => ({
     ...data,
+    status:
+      data.status === MERCHANT_REPORT_STATUSES_MAP.failed
+        ? MERCHANT_REPORT_STATUSES_MAP['quality-control']
+        : data.status,
```

Comment:
Fixed that failed reports won't appear as failed to the clients, they will be appear as "in quality control"